deployment/services/usage.ts

```
@@ -54,7 +54,7 @@ export function deployUsage({
         readinessProbe: {
           initialDelaySeconds: 10,
           periodSeconds: 5,
-          failureThreshold: 2,
+          failureThreshold: 1,
```

Fail fast. Avoid 503s